src/application-manager.coffee

```diff
@@ -398,57 +387,19 @@ module.exports = class ApplicationManager extends EventEmitter
 		targetNames = _.keys(target)
 		toBeRemoved = _.difference(currentNames, targetNames)
 		for name in toBeRemoved
-			outputPairs.push({
-				current: {
-					name
-					appId
-					config: current[name]
-				}
-				target: null
-			})
+			outputPairs.push({ current: current[name], target: null })
 		toBeInstalled = _.difference(targetNames, currentNames)
 		for name in toBeInstalled
-			outputPairs.push({
-				current: null
-				target: {
-					name
-					appId
-					config: target[name]
-				}
-			})
-		toBeUpdated = _.filter _.intersection(targetNames, currentNames), (name) =>
+			outputPairs.push({ current: null, target: target[name] })
+		toBeUpdated = _.filter _.intersection(targetNames, currentNames), (name) ->
 			# While we're in this in-between state of a network-manager, but not
```

Comment:
This comment is no longer needed :)

Reply:
It's like my brain just doesn't see outdated comments :P